uk.co.compucorp.civicrm.civihrsampledata/civihrsampledata.php

```diff
@@ -0,0 +1,116 @@
+<?php
```

Comment:
well we discussed that when you created leave and absence extension and we agreed to use :

**uk.co.compucorp.civicrm.ExtensionName** 

for extension names ( so I think it is fair to assume it is our new standard naming convention ) .

 Also this extension is  never used before so it is like a new extension so It is better to rename it to follow the other new ones.

also for the last part .. I think it reflect the extension name and it does not matter if it start with hr or not .. for example we have uk.co.compucorp.civicrm.appraisals which do not have the hr part ... I think the important here is to use uk.co.compucorp.civicrm. prefix and the last part can depend on the extension name .